app/javascript/src/utilities/utils.js

```diff
@@ -2,7 +2,8 @@
 import React from 'react'
 
 function CSRFToken ({win = window}: {win?: Window}) {
-  const getMetaContent = meta => win.document.head.querySelector(`meta[name~=${meta}][content]`).content
+  const hasCSRFParam = win.document.head.querySelector(`meta[name='csrf-param']`)
+  const getMetaContent = meta => hasCSRFParam ? win.document.head.querySelector(`meta[name~=${meta}][content]`).content : ''
```

Comment:
I'm not 100% sure about this approach... I'd like to hear what @hallelujah and @josemigallas thinks about it. An empty string won't make it work in the case it's not set by any  specific reason. It's good to avoid crashing, but maybe log the problem? leave it `undefined`... not entirely convinced